app/atoms.ts

```diff
@@ -0,0 +1,23 @@
+import { atomWithStorage, createJSONStorage, } from "jotai/utils";
+import { TwitchVideo, Video, VideoTest } from "~/interfaces";
+
+const videoListStorage = createJSONStorage<Video[]>(
```

Comment:
少しネストが深く、読みにくいので改善の余地があります。
例えば、早期 return することで、ネストを一段下げるべきです。
```suggestion
if (typeof value !== "object" || value === null)
```

Reply:
早期returnでリファクタリングしました。